polyply/tests/test_generate_templates.py

```diff
@@ -31,7 +31,9 @@
                                             _relabel_interaction_atoms,
                                             compute_volume, map_from_CoG,
                                             extract_block, GenerateTemplates,
-					    find_interaction_involving)
+					                        find_interaction_involving,
```

Comment:
indentlint

Reply:
curious in the text file it actually looks like it gas the correct indent already. let's see if removing and adding the line fixes the issue